src/ws-week-picker/ws-week-picker.js

```diff
@@ -0,0 +1,128 @@
+import {React, Component} from '../imports';
+import {WSWeekPickerCalendar} from './ws-week-picker-calendar';
+import './ws-week-picker.scss';
```

Comment:
please DON'T import the scss in the javascript. Import it from the scss files like demo/index.scss.